apprise/plugins/NotifyDBus.py

```diff
@@ -77,19 +83,26 @@
     NOTIFY_DBUS_SUPPORT_ENABLED = (
         LOOP_GLIB is not None or LOOP_QT is not None)
 
+    # ImportError: When using gi.repository you must not import static modules
+    # like "gobject". Please change all occurrences of "import gobject" to
+    # "from gi.repository import GObject".
+    # See: https://bugzilla.gnome.org/show_bug.cgi?id=709183
+    if "gobject" in sys.modules:  # pragma: no cover
+        del sys.modules["gobject"]
+
     try:
         # The following is required for Image/Icon loading only
         import gi
         gi.require_version('GdkPixbuf', '2.0')
         from gi.repository import GdkPixbuf
         NOTIFY_DBUS_IMAGE_SUPPORT = True
 
-    except (ImportError, ValueError, AttributeError):
+    except (ImportError, ValueError, AttributeError) as ex:
```

Comment:
I would ask that you role all of these exceptions on the module failing to load from `NotifyDBus`, `NotifyGnome`, and `NotifyMacOSX` so they are silent.  It's totally okay for these to fail.  Especially for Windows users.  They won't want to see a warning message everytime they use Apprise.

A global boolean is set that the module is disabled and this can be seen why in the `--details` for CLI users and `Apprise.details()` for API/Dev users.

Reply:
Now I understood your request. I think it will be fixed now?